cql/ManageSpecialPopulation.cql

```diff
@@ -330,8 +330,11 @@ define RecommendationForPatientsYoungerThan25:
       }  
     when ( // Heading4.1
       AgeInYears() < 25 and
-      Rare.MostRecentBiopsyReportWasWithinPastYear and
-      Rare.HistologyInterpretedAsCin3
+      Rare.AnyHistologyInterpretedAsCin3 and
+      (
+        Collate.DateOfLastCervicalPrecancerTreatment is null or 
+        not (Collate.DateOfLastCervicalPrecancerTreatment occurs after Collate.MostRecentCin3Biopsy.date)
+      )
```

Comment:
Kind of like in section I, it could be nice to create an expression called something like "NoTreatmentAfterCIN3", to be used here as well as in the rest of the expressions in K.1 Heading 4. Or better yet a function called "NoTreatmentAfterDate" that does the same thing but works for any date inputted as a parameter.

Reply:
Will update.

Reply:
Nice! Resolved in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/118/commits/c2704894cf2fdb29944fdfe2ff110d3d4094bda7. I think this makes things a lot more readable and repeatable.